src/lib/parquet/src/Flow/Parquet/ParquetFile/Data/RLEBitPackedHybrid.php

```diff
@@ -23,6 +23,12 @@ public function decodeBitPacked(BinaryReader $reader, int $bitWidth, int $varInt
             return;
         }
 
+        if ($bitWidth === 0) {
+            $output = \array_merge($output, \array_fill(0, \min($numGroups * 8, $maxItems), 0));
```

Comment:
we need to take `min` here, since from time to time we might need to pack less than 8 values and numGroups is always divisible by 8. So if we would pack zero 4 times, this would return us an array of 8 zeroes.